modules/apps/foundation/configuration-admin/configuration-admin-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -82,7 +86,7 @@ if (Validator.isNotNull(keywords)) {
 		total="<%= configurationModelIterator.getTotal() %>"
 	>
 		<liferay-ui:search-container-results
-			results="<%= configurationModelIterator.getResults(searchContainer.getStart(), searchContainer.getEnd()) %>"
+			results="<%= configurationModelIterator.getResults(searchContainer.getStart(), searchContainer.getEnd(), configurationModelNameComparator) %>"
```

Comment:
I find it weird that we create a getResults() method that receives a Comparator. Have you found other places in which we do that?
I think it's simpler to just get the results and do the desired shorting in the JSP invoking ListUtil ther.e